content/id/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Mengatur Service Account untuk Pod
+content_type: task
+weight: 90
+---
+
+<!-- overview -->
+Akun servis (_service account_) menyediakan identitas untuk proses yang sedang berjalan dalam sebuah Pod.
```

Comment:
no need to translate this since it's an API object
```suggestion
ServiceAccount menyediakan identitas untuk proses yang sedang berjalan dalam sebuah Pod.
```

Reply:
I'm not commenting for the same thing, but you can simply do replace all on:
* Service Account -> ServiceAccount
* _service account_ -> ServiceAccount
* _Service Account_ -> ServiceAccount
* service account -> ServiceAccount